Formula/chapel.rb

```diff
@@ -21,16 +21,18 @@ class Chapel < Formula
   # when Python dependency matches LLVM's Python for all OS versions.
   depends_on "python@3.9"
 
-  on_macos do
-    depends_on "llvm" if MacOS.version > :catalina
-    # fatal error: cannot open file './sys_basic.h': No such file or directory
-    # Issue ref: https://github.com/Homebrew/homebrew-core/issues/96915
-    depends_on "llvm@11" if MacOS.version <= :catalina
+  on_linux do
```

Comment:
Should this go last in the same way that `on_linux` blocks go after `on_macos` blocks?

Reply:
Yeah, probably...

Reply:
I opened a PR for this: https://github.com/Homebrew/brew/pull/13539